src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -45,6 +45,9 @@
         {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.returns.orderReturns|length }})
       </a>
     </li>
+
+    {{ renderhook('displayAdminOrderTabShip', {'id_order': orderForViewing.id}) }}
+    {{ renderhook('displayAdminOrderTabOrder', {'id_order': orderForViewing.id}) }}
```

Comment:
There is kind of BC break with these 2 hooks (and 2 related hooks below). In legacy, three params where passed to these hooks: `order` object model, order products array and `customer` object model. 

See https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/admin/AdminOrdersController.php#L1932

In new page, I pass only order id, but it's enough to retrieve all data about order (customer, products, order itself).

@matks wdty? Is it ok or should we keep legacy params?

Reply:
Calling some friends developing modules on this one 😄 
@v4lux @apacios @Quetzacoalt91 wdyt ? is it better to keep as much as possible previous hook payload, or is it better to make it faster and lighter through the migration ?

Reply:
I can see the interest to have "smaller" hooks, but if the previous hooks had those three variables we' shouldn't change them Besides what would appear as a performance gain (only give the order id) might have drawbacks in the end (each module would have to request the same order)